src/rpc/misc.cpp

```diff
@@ -151,6 +155,13 @@ static RPCHelpMan createmultisig()
     result.pushKV("redeemScript", HexStr(inner));
     result.pushKV("descriptor", descriptor->ToString());
 
+    UniValue warnings(UniValue::VARR);
+    std::optional<OutputType> created_output_type = OutputTypeFromDestination(dest);
+    if (!created_output_type || created_output_type.value() != output_type) {
```

Comment:
Might check `!request.params[2].isNull()` here too, in case a future PR changes the default address type?